src/main/java/edu/ksu/canvas/requestOptions/SubmissionOptions.java

```diff
@@ -0,0 +1,124 @@
+package edu.ksu.canvas.requestOptions;
+
+import java.util.Map;
+
+/**
+ * Rubric_Assessment not supported yet.
+ */
+public class SubmissionOptions extends BaseOptions {
+
+    public class StudentSubmissionOption {
```

Comment:
I prefer to put inner classes at the bottom of the file. When reading this class it is easy to get confused and think the private fields belong to the outer class. Putting the inner class at the bottom of the file moves it out of the way unless you actually **need** to look at it.